test/unit/rope_out_of_memory.result

```diff
@@ -0,0 +1,503 @@
+	*** out_of_memory_stress_test ***
+1..500
```

Comment:
Could you please reduce result output? 500 checks is too much imho

Reply:
reduced to 20